composition/content_merge.go

```diff
@@ -55,12 +58,20 @@ func NewContentMerge(metaJSON map[string]interface{}) *ContentMerge {
 	return cntx
 }
 
+func (cntx *ContentMerge) SetDeduplicationStrategy(strategy StylesheetDeduplicationStrategy) {
+	cntx.stylesheetDeduplicationStrategy = strategy
+}
+
 func (cntx *ContentMerge) collectStylesheets(f Fragment) {
 	cntx.stylesheets = append(cntx.stylesheets, f.Stylesheets()...)
 }
 
 func (cntx *ContentMerge) writeStylesheets(w io.Writer) {
-	for _, attrs := range cntx.stylesheets {
+	stylesheets := cntx.stylesheets
```

Comment:
Because in the following lines I set this variable to the result of the deduplication algorithm, if one is given.